charts/brigade-bitbucket-gateway/values.yaml

```diff
@@ -44,9 +44,11 @@ tls:
   # key: base 64 encoded key goes here
 
 ingress:
-  ## Whether to enable ingress. By default, this is disabled and the gateway's
-  ## service is of type LoadBalancer instead. Enabling ingress is advanced
```

Comment:
This wasn't entirely true. Yes -- ingress is disabled by default, but no -- the service type is actually `ClusterIP` by default and has been for quite some time. We had just overlooked updating this comment.